packages/desktop-client/src/components/reports/SaveReport.tsx

```diff
@@ -145,15 +160,14 @@ export function SaveReport({
             flexShrink: 0,
           }}
         >
-          {!report.id ? 'Unsaved report' : report.name}&nbsp;
+          {report.id === '' ? 'Unsaved report' : report.name}&nbsp;
```

Comment:
Did we need this to change? The previous covered all falsey including empty string.